modules/cleanup-a-backup-oadp-rosa-sts.adoc

```diff
@@ -81,13 +88,14 @@ $ aws s3 rm s3://${CLUSTER_NAME}-oadp --recursive
 ----
 $ aws s3api delete-bucket --bucket ${CLUSTER_NAME}-oadp
 ----
+
 . Detach the policy from the role by running the following command:
 +
 [source,terminal]
 ----
-$ aws iam detach-role-policy --role-name "${ROLE_NAME}" \
-  --policy-arn "${POLICY_ARN}"
+$ aws iam detach-role-policy --role-name "${ROLE_NAME}" \ --policy-arn "${POLICY_ARN}"
```

Comment:
Do you still need the `\` now that the command is on one line? That character is when the command is split across multiple lines, right?

Reply:
```suggestion
$ aws iam detach-role-policy --role-name "${ROLE_NAME}"  --policy-arn "${POLICY_ARN}"
```